### DISCUSSION TITLE
Some questions about vehicles

### DISCUSSION THREAD

**Discussion by HD6630** [YimMenu/YimMenu#3632] [General]
These issues have been bothering me for a long time. After generating the vehicle, I can only use the machine gun in the vehicle. When using missiles, the missiles cannot be launched and explode like fireworks stuck in the launch well on my vehicle, but there is no damage. Now I cannot use any vehicles with missiles in GTAOL, not even my own RPG, and can only stay away from my vehicle. And these bugs only occur after I inject YimMenu. What's going on?

**Comment by xesdoog** [comment]
Are you using [this script](https://github.com/YimMenu-Lua/Samurais-Scripts)? If so then that's the '**Missile Defense**' option that can be found under the vehicle tab. It removes/destroys any missiles near you vehicle including ones fired by you.

**Reply by HD6630** [reply]
oh thanks,I solved it with your method
